src/components/MainContainer/MainContainer.jsx

```diff
@@ -46,6 +47,8 @@ class MainContainer extends React.Component {
                         films={this.props.films}
                         genres={this.props.genres}
                     />
+                ) : this.props.loading ? (
```

Comment:
It will be better to avoid this construction. As an option, you can write 
`render() {
if(this.props.loading) {
return <LoaderComponent />
}
return (
....
)
`

Reply:
Fixed